chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -218,6 +218,7 @@
 
         when os =~ /^(open)?suse/
           append << "install=#{install_url} autoyast=#{node_url}/autoyast.xml"
+          append << "ifcfg=dhcp4 netwait=60"
```

Comment:
I understand we do this for SLE12. This doesn't have any downside on SLE11 (ie, this still works fine there)?